docs/UserGuide.adoc

```diff
@@ -212,13 +208,13 @@ Displays details of policy at index 1 on the right panel.
 === Editing a person : `edit`
 
 Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [ic/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [dob/DATE_OF_BIRTH]`
+Format: `edit INDEX [n/NAME] [ic/NRIC] [p/PHONE] [e/EMAIL] [a/ADDRESS] [dob/DATE_OF_BIRTH] [g/GENDER]`
 
 ****
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
-* If editing a person's date of birth results in him being ineligible for a policy/multiple policies he currently possesses, they will be unassigned from him.
+* If editing a person's details results in him being ineligible for a policy/multiple policies he currently possesses, they will be unassigned from him.
 ****
```

Comment:
Can we standardise and use sample outputs? 

UG/ good use of examples: Uses examples e.g., sample inputs/outputs.